bip-0009/assignments.mediawiki

```diff
@@ -18,11 +18,11 @@ State can be defined, active, failed. Dates are in UTC.
 | csv
 | 0
 | 2016-05-01 00:00:00
-| 2017-05-01 00:00:00 
+| 2017-05-01 00:00:00
 | defined
 | 2016-03-01 00:00:00
 | 2017-05-01 00:00:00
-| active since #770111
+| active since #770112
```

Comment:
I would tend to agree that it's #770112 because it's the first block that enforces the new rules.